dev/custom-pages/orlandounited.html

```diff
@@ -74,94 +66,55 @@ <h2 class="section-heading" style="margin-top: 30px;">Thank You Blood Donors</h2
+  UCF has resources available to all who need them.
 
   <ul>
-    <li><a href="http://cares.sdes.ucf.edu">UCF Cares</a></li>
-    <li><a href="http://victimservices.ucf.edu/" target="_blank">UCF Victim Services</a></li>
+    <li><a href="https://caps.sdes.ucf.edu/" target="_blank">Counseling and Psychological Services</a>
+    <ul style="margin-bottom:0;margin-top:5px;"><li><a href="http://victimservices.ucf.edu/" target="_blank">UCF Victim Services</a></li></ul>
+    </li>
+    <li><a href="http://cares.sdes.ucf.edu/" target="_blank">UCF Cares</a></li>
+    <li><a href="http://sciences.ucf.edu/psychology/ucf-restores/" target="_blank">UCF Restores</a></li>
   </ul>
 
-  <div class="row">
-    <div class="col-md-5">
-     <h4>UCF RESTORES:</h4>
-
-      <strong>Daily</strong> at UCF's <a href="http://psychology.cos.ucf.edu/ucf-restores/" target="_blank">Center for RESearch and Treatment On Response to Extreme Stressors</a>, <a href="http://map.ucf.edu/?show=99" target="_blank">Psychology Building 139</a>. Free and open to anyone in the community who needs assistance.
-    </div>
-    <div class="col-md-5 col-md-offset-1">
-      <h4>Employee Assistance Program Resources:</h4>
-
-      This tragedy has impacted our staff and faculty members. Employees can find more about <a href="https://hr.ucf.edu/eap-support-for-orlando-shooting/" target="_blank">UCF’s Employee Assistance Program here.</a>
-    </div>
-  </div>
-
-  Off campus, the <a href="http://www.thecenterorlando.org/">GLBT Community Center of Orlando</a> is working with community partners to provide crisis services to the families, friends and others affected by the shooting. Other community partners offering assistance are <a href="http://zebrayouth.org/" target="_blank">Zebra Coalition</a> and <a href="http://www.eqfl.org/" target="_blank">Equality Florida</a>.
-
+  Off campus community partners include the <a href="http://www.thecenterorlando.org/" target="_blank">GLBT Community Center of Orlando</a>, <a href="http://www.zebrayouth.org" target="_blank">Zebra Coalition</a> and <a href="http://www.eqfl.org/" target="_blank">Equality Florida</a>.
 </section>
 
-<hr>
+<hr/>
 
 <section>
-  <h2 class="section-heading" style="margin-top: 45px;">Other Resources and Information</h2>
-  <ul class="list-navigation list-unstyled">
-    <li><a href="http://www.oneorlando.org/" target="_blank">OneOrlando Fund</a></li>
-    <li><a href="http://www.cityoforlando.net/blog/orlandounited-want-to-help-let-us-know/" target="_blank">City of Orlando Volunteer Information</a></li>
-  </ul>
+  <h2 class="section-heading" style="margin-top: 20px;">Pulse Anniversary News</h2>
+
+  [ucf-news-feed topics="pulse" title="" layout="modern"]
```

Comment:
We just need to make sure this layout is available in production before moving this content up.